hal/src/sleeping_delay.rs

```diff
@@ -49,6 +81,7 @@ where
         let mut count: u32 = 1 + (us / NUM_US_IN_S);
 
         // Start the timer and sleep!
+        // TODO: why does this use nanos?
         self.timer.start((us / count).nanos());
```

Comment:
I must not understand fugit correctly, but it looks to me like this is treating microseconds as nanos with no conversion. What is going on here?